### DISCUSSION TITLE
A beginner's guide to Event Storming

### DISCUSSION THREAD

**Discussion by NTCoding** [ddd-crew/ddd-starter-modelling-process#58] [Team Posts]
Hey everyone,

One of the requests I commonly receive is how to get started with Event Storming. Something that helps people who have seen the idea and want to try it but need help running their first workshop.

I know we have the glossary repo which contains the workshop process but it's quite minimal and I think a bit more detail would be useful for beginners.

We could create a new repo or continue to work in the existing repo or link to existing (free) resources that already cover this. Maybe we can add a link at the top of the Glossary repo to whatever the final solution is as well.

Look forward to hearing your thoughts.

**Comment by yellowbrickc** [comment]
I would opt for extending the existing repo with more details and additional resources (I can't think of any reason how a new repo would help to get the information someone is looking for faster and clearer 🤔 )

**Comment by NTCoding** [comment]
The reason I mentioned the option of a different repo is that adding a beginners guide stretches the definition of the current name. i.e. it's not really a glossary or a cheat sheet anymore. What do you think -- would you keep the current name?

**Comment by yellowbrickc** [comment]
Now I understand 👍. It is a good question. I still would add it to the ES-glossary repo or to the [ddd-starter-modelling-process](https://github.com/ddd-crew/ddd-starter-modelling-process). We already have a lot of references to domain modelling tools, we won't need a new one for collaborative modelling, isn't it? 

I suggest using the ddd-starter-modelling-process: it has 3.3k stars compared to the ES-repo having 500. Adding it to that repo will have the biggest impact.

**Comment by yellowbrickc** [comment]
after that, we can (and should) reference it from the ES-repo.
